### DISCUSSION TITLE
how to upgrade jans server ?

### DISCUSSION THREAD

**Discussion by manojs1978** [JanssenProject/jans#1081] [Q&A] [answered]
If jans server is already installed and we need to apply some updates or apply some patches. how to apply it.

**Comment by manojs1978** [comment, marked as answer]
**jans upgrade ubuntu OS **
1) download latest package file 

     wget http:$(curl -s -L https://api.github.com/repos/JanssenProject/jans/releases/latest | egrep -o '/.*ubuntu20.04_amd64.deb' | head -n -O /tmp/jans.ubuntu20.04_amd64.deb

2) run below upgrade command 

   sudo apt upgrade -y /tmp/jans.ubuntu20.04_amd64.deb

3) verify upgraded version

apt-cache policy jans

**jans upgrade on RHEL/CentOS8**
1) download and upgrade latest package file 

yum upgrade -y https:$(curl -s -L https://api.github.com/repos/JanssenProject/jans/releases/latest | egrep -o '/.*el8.x86_64.rpm' | head -n 2) verify upgraded version

rpm -qa |grep jans

**Reply by ossdhaval** [reply]
Hey @manojs1978 @moabu Can these steps be part of the [upgrade section](https://docs.jans.io/head/admin/upgrade/) in Janssen documentation?

**Reply by manojs1978** [reply]
yes , we can include this steps.
cc: @devrimyatar @moabu

**Comment by ossdhaval** [comment]
@manojs1978 , as discussed, let's move this to documentation, under [VM upgrade](https://docs.jans.io/v1.0.9/admin/upgrade/vm/)

**Reply by manojs1978** [reply]
done
